src/libNode/FinalBlockProcessing.cpp

```diff
@@ -427,6 +427,10 @@ void Node::CallActOnFinalblock() {
 
   LOG_MARKER();
 
+  if (m_mediator.m_ds->m_mode == DirectoryService::IDLE) {
```

Comment:
Or can just remove the call to `CallActOnFinalblock` inside `DirectoryService::ProcessFinalBlockConsensusWhenDone`?

Reply:
fixed